src/libcollections/vec_deque.rs

```diff
@@ -84,6 +84,7 @@ impl<T> Drop for VecDeque<T> {
 
 #[stable(feature = "rust1", since = "1.0.0")]
 impl<T> Default for VecDeque<T> {
+    /// Creates a `VecDeque<T>` with a constant initial capacity.
```

Comment:
This makes little sense. Should probably say the same thing as `VecDeque::new`, which is “Creates a new empty `VecDeque”.